mne/io/base.py

```diff
@@ -1702,25 +1705,43 @@ def _allocate_data(data, data_buffer, data_shape, dtype):
     return data
 
 
-def _time_as_index(times, sfreq, first_samp=0, use_first_samp=False):
+def _time_as_index(times, sfreq, first_samp=0, use_first_samp=False,
+                   use_rounding=False):
     """Convert time to indices
 
     Parameters
     ----------
     times : list-like | float | int
         List of numbers or a number representing points in time.
+    sfreq : float | int
+        Sample frequency.
+    first_samp : int
+       Index to use as first time point.
     use_first_samp : boolean
         If True, time is treated as relative to the session onset, else
         as relative to the recording onset.
+    use_rounding : boolean
+        If True, use rounding (instead of truncation) when converting times to
+        indicies. This can help avoid non-unique indices.
 
     Returns
     -------
     index : ndarray
         Indices corresponding to the times supplied.
+
+    Notes
+    -----
+    np.around will return the nearest even number for values exactly between
+        two integers.
     """
     index = np.atleast_1d(times) * sfreq
     index -= (first_samp if use_first_samp else 0)
-    return index.astype(int)
+
+    # Round or truncate time indices
+    if use_rounding:
+        return np.around(index).astype(int)
```

Comment:
They're the same function, but I can easily change to `np.round` if it's clearer.